### PR TITLE
fix: {{user}} tags in templates/examples empty when passed to LLM

### PR DESCRIPTION
**Pull request by tcm390**
related: https://github.com/ai16z/eliza/issues/1267

截圖 2024-12-20 下午11 08 14

### CODE REVIEW

File: packages/core/src/context.ts [hunk 1]
```diff
@@ -69,3 +70,35 @@ export const composeContext = ({
...
+ *
+ * @example
+ * // Given a template and a length
+ * const template = "Hello, {{user1}}! Meet {{user2}} and {{user3}}.";
+ * const length = 3;
+ *
+ * // Composing the random user string will result in:
+ * // "Hello, John! Meet Alice and Bob."
+ * const result = composeRandomUser({ template, length });
+ */
```
Comment (monilpat) [inline packages/core/src/context.ts:93]: Can this be part of a README somewhere?

**Review by monilpat — DISMISSED** [review]
LGTM outside of comment :)

File: packages/core/src/context.ts (hunk 1 above)
Comment (odilitime) [reply packages/core/src/context.ts:93]: nothing there belongs in the readme, just looks like an LLM being too verbose. It's all dev facing stuff

**Review by monilpat — APPROVED** [review]
LGTM

**Pull request merged**